### PR TITLE
Updated the JetStream Deployment Guide 13.1

### PR DESCRIPTION
**Pull request by WajeehaHamid**
Deployment guide for 13.1 release has been updated. Kindly Review
@MMP1 
@heydog15

### CODE REVIEW

File: doc/deployment_guide.md [hunk 1]
```diff
@@ -49,5422 +49,5841 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
...
+### General Hardware Options
+
+To reduce time spent on specifying hardware for an initial system, the Architecture Guide offers a full solution using validated Dell EMC PowerEdge server hardware designed to allow a wide range of configuration options, including optimized configurations for: 
+
+* Compute nodes
+* Infrastructure nodes
+* Storage nodes
+
+Dell EMC recommends starting with OpenStack software using components from this Architecture Guide -
+Version 13.1 because the hardware and operations processes comprise a flexible foundation upon which to
```
Comment (MMP1) [inline doc/deployment_guide.md]: Replace lines 83-84 with:

Version 13.1 flexible foundation expands as your cloud deployment grows, which protects investments.

File: doc/deployment_guide.md [hunk 2]
```diff
@@ -49,5422 +49,5841 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
...
+expand as your cloud deployment grows, so your investment is protected.
+
+As noted throughout this Architecture Guide - Version 13, Dell EMC constantly adds capabilities to expand
```
Comment (MMP1) [inline doc/deployment_guide.md]: Replace line 86-87 with:
Dell EMC Architecture Guide version 13 expands hardware capabilities.

File: doc/deployment_guide.md [hunk 3]
```diff
@@ -49,5422 +49,5841 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
...
+
+**Dell EMC PowerEdge R740/Dell EMC PowerEdge R740xd servers**
+
+The Dell EMC PowerEdge R740/Dell EMC PowerEdge R740xd delivers a perfect balance between storage scalability and performance. The 2U two-socket platform is ideal for software defined storage. The R/740/R740xd versatility is highlighted with the ability to mix any drive type to create the optimum configuration of SSD and HDD for either performance, capacity or both.
+
+> Note: The Dell EMC PowerEdge R740/Dell EMC PowerEdge R740xd is the platform of choice for software defined storage and is the foundation for Red Hat Ceph Storage
```
Comment (MMP1) [inline doc/deployment_guide.md]: Add a period:
"...nd is the foundation for Red Hat Ceph Storage."

File: doc/deployment_guide.md [hunk 4]
```diff
@@ -49,5422 +49,5841 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
...
+
+Network configuration is based upon using the Neutron-based options supported by the RHOSP code base, and does not rely upon third-party drivers. This reference configuration is based upon the Neutron networking services using the ML2 drivers for Open vSwitch with the VLAN option.
+
+#### Networking includes:
+
+* Core and layered networking capabilities
+* Network Function Virtualization (NFV)
+* 25GbE 100GbE networking
+* NIC bonding
+* Redundant trunking top-of-rack (ToR) switches into core routers
```
Comment (MMP1) [inline doc/deployment_guide.md]: Add a period:
"... switches into cord routers."

File: doc/deployment_guide.md [hunk 5]
```diff
@@ -49,5422 +49,5841 @@ Red Hat®, Red Hat Enterprise Linux®, and Ceph are trademarks or registered tra
...
+> Note: Please contact your Dell EMC sales representative for a detailed parts list or contact Dell EMC Professional Services team for OpenStack using the email [rhosp.ra.ps@emc.com](malito:rhosp.ra.ps@emc.com)
+
+### Version 13.1 features
+
+**The following features are added in the JetPack Automation and the Ready Architecture for RedHat OpenStack:**
+
+* Support for the lastest release of Red Hat OpenStack Platform 13.1 including the latest updates.
+* Support for latest release of RHEL 7.6 including the latest updates.
+* Support for Red Hat Ceph Storage version 3.1
+* Added support for jumbo frames.
```
Comment (MMP1) [inline doc/deployment_guide.md]: Replace:
"jumbo frames."
with
"Jumbo Frames."

File: doc/deployment_guide.md (hunk 1 above)
Comment (WajeehaHamid) [reply doc/deployment_guide.md]: I have resolved all the recommended changes.

Comment (MMP1) [reply doc/deployment_guide.md]: Completed review.

**Pull request merged**